### PR TITLE
bump to 2.3

### PR DESCRIPTION
**Pull request by rbhavna**
Signed-off-by: Bhavana Ramaram <rbhavna@amazon.com>

### Description
Bumped opensearch version to 2.3
Upgrade to lucene snapshot
 
### Issues Resolved
[List any issues this PR will resolve]
 
### Check List
- [ ] New functionality includes testing.
  - [ ] All tests pass
- [ ] New functionality has been documented.
  - [ ] New functionality has javadoc added
- [x] Commits are signed per the DCO using --signoff 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Review by Zhangxunmt — APPROVED** [review]

**Review by ylwu-amzn — DISMISSED** [review]

File: build.gradle [hunk 1]
```diff
@@ -9,7 +9,7 @@ buildscript {
     ext {
         opensearch_group = "org.opensearch"
         isSnapshot = "true" == System.getProperty("build.snapshot", "true")
-        opensearch_version = System.getProperty("opensearch.version", "2.2.0-SNAPSHOT")
+        opensearch_version = System.getProperty("opensearch.version", "3.0.0-SNAPSHOT")
```
Comment (ylwu-amzn) [inline build.gradle:12]: Sorry, realize the branch is 2.x after approving. Can you change it to `3.0.0-SNAPSHOT` ?

Comment (rbhavna) [reply build.gradle:12]: Already changed it to 3.0.0. Does it need to be 2.3.0-SNAPSHOT?

**Comment by rbhavna** [comment]
When bumped version to 2.3, the following error showed up due to a dependency issue. It should be resolved once the 'common-utils' team bump their version.

Screen Shot 2022-08-19 at 12 18 34 PM

**Review by ylwu-amzn — APPROVED** [review]

**Comment by ylwu-amzn** [comment]
hi, @rbhavna auto PR #414 bumped version to 2.3. Can you close this PR and create a new PR with `build-tools/repositories.gradle` change?

**Comment by rbhavna** [comment]
okay @ylwu-amzn

**Pull request closed without merging**